src/components/services/LoanManagerService.ts

```diff
@@ -24,7 +24,11 @@ const USD_BASED_TOKEN_IDS: string[] = [
   // USDC on Ethereum
   'a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
   // USDC on Polygon
-  '2791Bca1f2de4661ED88A30C99A7a9449Aa84174'
+  '2791Bca1f2de4661ED88A30C99A7a9449Aa84174',
+  // USDT on Polygon
+  '0xdAC17F958D2ee523a2206206994597C13D831ec7',
+  // DAI on Polygon
+  '0xdAC17F958D2ee523a2206206994597C13D831ec7'
```

Comment:
wrong contract for DAI. should be 

`0x6B175474E89094C44Da98b954EedeAC495271d0F`